htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxResponse.java

```diff
@@ -205,8 +205,8 @@ public HtmxResponse and(HtmxResponse otherResponse) {
         if (otherResponse.getHeaderRefresh()) {
             this.headerRefresh = true;
         }
-        if (otherResponse.getHeaderRedirect() != null) {
-            this.headerRedirect = otherResponse.getHeaderRedirect();
+        if (otherResponse.getHeaderRetarget() != null) {
+            this.headerRetarget = otherResponse.getHeaderRetarget();
```

Comment:
This is the main change of the PR. (Couldn't find it in the reformatting) Thanks for splitting the commits.

Reply:
I would always review commit by commit, not the whole PR diff.